src/ralph/ui/forms/devices.py

```diff
@@ -397,6 +404,16 @@ class Meta(DeviceForm.Meta):
             'deleted',
         )
 
+    def set_asset_status(self):
+        asset_status = ''
+        linked_asset = self.instance.get_asset()
+        if linked_asset:
+            asset_status = (
+            AssetStatus.NameFromID(linked_asset.status) if
```

Comment:
afaik there is underscore version of this method (`name_from_id`), which is more pythonic - could you use it?